bin/nrnivmodl_makefile_cmake.in

```diff
@@ -5,6 +5,7 @@ MECH_NAME =
 MECH_VERSION = 0.0
 MODS_PATH = .
 OUTPUT = @CMAKE_HOST_SYSTEM_PROCESSOR@
+OUTPUT = .
```

Comment:
I did so along with a couple others.  Here is a detail
```
INCLUDES = -I. -I.. ...
```
Should I get rid of those. I left the ```-I..``` in  because the c translated mod file is in ```./``` and in the most typical case the mod file is in ```../```.  So it seems slightly safer because the added ```-I<mod_dir_path> is at the end of the includes. (Being  after COMPILE as I didn't take the trouble to divide
```
COMPILE = $(CC) $(CFLAGS) @NRN_COMPILE_DEFS@ $(INCLUDES)
```
in order to be able to put it before the INCLUDES. I guess I should divide it so that it is possible to effectively prepend to INCLUDES.)